.github/workflows/scheduled.yml

```diff
@@ -45,6 +45,7 @@ jobs:
     container:
       image: ghcr.io/homebrew/ubuntu22.04:master
     needs: create_matrix
+    continue-on-error: true
```

Comment:
This just means your jobs will always succeed regardless of the exit code.

Reply:
Should it be on a step level then?

Reply:
No, it should be `fail-fast: false` under `strategy:`

Reply:
You can also set it at a step level, but that step will show up as 🟢 regardless of what happens there